hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/ReconUtils.java

```diff
@@ -385,4 +396,61 @@ public SCMNodeDetails getReconNodeDetails(OzoneConfiguration conf) {
         HddsServerUtil.getReconDataNodeBindAddress(conf));
     return builder.build();
   }
+
+  /**
+   * This method accepts the list of command args and supports execution of ozone CLI based commands only.
+   *
+   * @param commandArgs the list of command args
+   * @return the command output map with its exit code
+   */
+  public Map<Integer, String> executeCommand(List<String> commandArgs) {
+    String command = joinListArgs(commandArgs);
+    Map<Integer, String> processOutputMap = new HashMap<>();
+    LOG.info("Received command : '{}' to execute", command);
+    AtomicReference<String> processOutput = new AtomicReference<>();
+    ProcessBuilder pb = new ProcessBuilder(commandArgs);
+    try {
+      SecurityUtil.doAsLoginUser((PrivilegedExceptionAction<Map<Integer, String>>) () -> {
+        Process process = pb.start();
+        int exitCode = process.waitFor();
```

Comment:
we may need provide max time for it to wait

Reply:
Ok, sure.